Solita.LocalizationEditor.UI/Business/BlobLocalizationProviderInitiator.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using EPiServer.Core;
+using EPiServer.Framework;
+using EPiServer.Framework.Initialization;
+using EPiServer.Framework.Localization;
+using EPiServer.Events.Clients;
+using System.Collections.Specialized;
+using Solita.LocalizationEditor.UI.Services;
+using EPiServer.Logging.Compatibility;
+using EPiServer;
+using EPiServer.ServiceLocation;
+using EPiServer.Events;
+
+namespace Solita.LocalizationEditor.UI.Business
+{
+
+    [InitializableModule]
+    //[ModuleDependency(typeof(FrameworkInitialization))]
+    [ModuleDependency(typeof(EPiServer.Web.InitializationModule))] 
+    public class BlobLocalizationProviderInitiator : IInitializableModule
+    {
+        private const string ProviderName = "Translations";
+        private static readonly ILog Log = LogManager.GetLogger(typeof(BlobLocalizationProviderInitiator));
+        private static bool initialized;
+        private ProviderBasedLocalizationService localizationService;
```

Comment:
Couple of coding style notes: The commented code line (ModuleDependency) should be removed and the suggested names for "initialized" & "localizationService" would be "_initialized" & "_localizationService"